node_filesystem/templates/bird.cfg.template

```diff
@@ -63,20 +75,40 @@ template bgp bgp_template {
   graceful restart;  # See comment in kernel section about graceful restart.
 }
 
-{{if ls "/config/bgp_peer_rr_v4"}}
-# Peering with configured neighbors.
-{{range gets "/config/bgp_peer_rr_v4/*"}}{{$nums := split .Value "."}}{{$id := join $nums "_"}}
-# For peer {{.Value}}
+{{/* ------------- Node-to-node mesh ------------- */}}
+{{if lsdir "/host"}}
```

Comment:
hmmm i think previous testing indicated that it was required, but checking calicoctl we do set up etcd before creating the node so we should be fine....

and i've just retested without it, so I guess its ok.  I'll remove.